### ISSUE TITLE
Literature references

### ISSUE THREAD

**Issue by coreytcallaghan** [DrJoseValdez/urban_species#1]
A thread to keep track for references

https://onlinelibrary.wiley.com/doi/abs/10.1111/geb.12404

**Comment by coreytcallaghan** [comment]
R package that uses IUCN distribution to calculate EOO and AOO. https://bdj.pensoft.net/articles.php?id=20530

**Comment by coreytcallaghan** [comment]
https://link.springer.com/article/10.1007/s10531-020-02088-1

**Comment by DrJoseValdez** [comment]
> 
> 
> R package that uses IUCN distribution to calculate EOO and AOO. https://bdj.pensoft.net/articles.php?id=20530

Yes! I sent you that yesterday, I'll try to have a look and play around with it for a bit.
